package.json

```diff
@@ -18,6 +18,9 @@
     "build/index.js",
     "build/example.js"
   ],
+  "dependencies": {
+      "prop-types": "^15.5.10"
```

Comment:
I'm trying to avoid any dependencies in this library, since this is a simple icon lib. 

I don't think it's too much of a burden for users of this lib to have React and PropTypes as dependencies. This is why when we update the package to use prop-types in our component, we'll need to bump the major version since the requirements have changed.